transactionoutbox-core/src/main/java/com/gruelbox/transactionoutbox/DefaultInvocationSerializer.java

```diff
@@ -125,41 +128,44 @@ public Invocation deserializeInvocation(Reader reader) {
       addClassPair(boolean.class, "boolean");
       addClassPair(char.class, "char");
 
-      addClassPair(Byte.class, "Byte");
-      addClassPair(Short.class, "Short");
-      addClassPair(Integer.class, "Integer");
-      addClassPair(Long.class, "Long");
-      addClassPair(Float.class, "Float");
-      addClassPair(Float.class, "Float");
-      addClassPair(Float.class, "Float");
-      addClassPair(Double.class, "Double");
-      addClassPair(Boolean.class, "Boolean");
-      addClassPair(Character.class, "Character");
-
-      addClassPair(BigDecimal.class, "BigDecimal");
-      addClassPair(String.class, "String");
-      addClassPair(Date.class, "Date");
-
-      addClassPair(DayOfWeek.class, "DayOfWeek");
-      addClassPair(Duration.class, "Duration");
-      addClassPair(Instant.class, "Instant");
-      addClassPair(LocalDate.class, "LocalDate");
-      addClassPair(LocalDateTime.class, "LocalDateTime");
-      addClassPair(Month.class, "Month");
-      addClassPair(MonthDay.class, "MonthDay");
-      addClassPair(Period.class, "Period");
-      addClassPair(Year.class, "Year");
-      addClassPair(YearMonth.class, "YearMonth");
-      addClassPair(ZoneOffset.class, "ZoneOffset");
-      addClassPair(DayOfWeek.class, "DayOfWeek");
-      addClassPair(ChronoUnit.class, "ChronoUnit");
-
-      addClassPair(Transaction.class, "Transaction");
-      addClassPair(TransactionContextPlaceholder.class, "TransactionContext");
+      addClass(Byte.class);
+      addClass(Short.class);
+      addClass(Integer.class);
+      addClass(Long.class);
+      addClass(Float.class);
+      addClass(Double.class);
+      addClass(Boolean.class);
+      addClass(Character.class);
+
+      addClass(BigDecimal.class);
+      addClass(String.class);
+      addClass(Date.class);
+      addClass(UUID.class);
+
+      addClass(DayOfWeek.class);
+      addClass(Duration.class);
+      addClass(Instant.class);
+      addClass(LocalDate.class);
+      addClass(LocalDateTime.class);
+      addClass(Month.class);
+      addClass(MonthDay.class);
+      addClass(Period.class);
+      addClass(Year.class);
+      addClass(YearMonth.class);
+      addClass(ZoneOffset.class);
+      addClass(DayOfWeek.class);
+      addClass(ChronoUnit.class);
+
+      addClass(Transaction.class);
+      addClass(TransactionContextPlaceholder.class);
```

Comment:
That differs from `addClassPair(TransactionContextPlaceholder.class, "TransactionContext");`, is it ok?

Reply:
Good spot! I went through them and failed on the last line. I will undo that - I don't have an interest in accidentally altering behaviour here - the old name might be persisted.

Reply:
Fixed. Reverted to the old name for this one.